interpreter/CMakeLists.txt

```diff
@@ -482,6 +482,9 @@ if (builtin_cling)
     get_directory_property(LLVM_DEFS DIRECTORY llvm-project/llvm COMPILE_DEFINITIONS)
     # Turns DEFINE1;DEFINE2 to -DDEFINE1 -DDEFINE2
     string (REPLACE ";" " -D" LLVM_DEFS ";${LLVM_DEFS}")
+
+    # Turn off coverage - we don't need this for llvm.
+    string(REPLACE "${GCC_COVERAGE_COMPILE_FLAGS}" "" CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS}")
```

Comment:
Did you test this? We are calling `add_subdirectory(llvm-project/llvm)` approximately 200 lines above, I don't see how adding this statement here can help...